bootstrap/common.sh

```diff
@@ -3,6 +3,9 @@
 # Util functions cloud reusable.
 APOLLO_ROOT=$(dirname "${BASH_SOURCE}")/..
 DEFAULT_CONFIG="${APOLLO_ROOT}/bootstrap/${APOLLO_PROVIDER}/${APOLLO_CONFIG_FILE-"config-default.sh"}"
+# It must be change before mergeing with capgemini master branch
+DYNAMIC_INVENTORY="https://raw.githubusercontent.com/ravbaba/terraform.py/azure-provider/terraform.py"
```

Comment:
The url should be change back to https://raw.githubusercontent.com/Capgemini/terraform.py/master/terraform.py after merging my PR for terraform.py repo